packages/libllvm/build.sh

```diff
@@ -96,16 +89,21 @@ termux_step_pre_configure() {
 		termux_error_exit "Invalid arch: $TERMUX_ARCH"
 	fi
 	# see CMakeLists.txt and tools/clang/CMakeLists.txt
-	TERMUX_PKG_EXTRA_CONFIGURE_ARGS+=" -DLLVM_DEFAULT_TARGET_TRIPLE=$LLVM_DEFAULT_TARGET_TRIPLE"
```

Comment:
[Set to `LLVM_HOST_TRIPLE` by default](https://github.com/llvm/llvm-project/blob/d4579b7ef696dc932f3174896ee5de7d235c6462/llvm/CMakeLists.txt#L647), so need to set it separately.

Reply:
Host and target are different. They have to be explicitly set as the same. This is due to jit compiler. Last time it wasn't set like that jit compiled and tried to run x86_64 code on aarch64. It then fails...

Reply:
Not sure what you mean, the host triple is also set to this just two lines below. The LLVM CMake config then sets the default target triple to be the same as the host triple on Android.

Reply:
When using jit for say julialang it first v

Reply:
Jit compiles code then runs because the default target and host can be different, say default target is i686 and the host is x86_64. This means jit will use x86_64 and because it needs to run on host. Jit code is more like a script that then compiles it first time you run it. 
For us last time the build host remained default target were different causing the jit to use x86_64. Since that was the host it was built on.

Reply:
I get that, but [the host triple is set to Android just two lines below](https://github.com/termux/termux-packages/pull/5136/files#diff-fa5845060496857161950d6dca5e1ba3R93), and then LLVM sets the default target triple to the host triple in the code I linked in my original comment, so they should always be the same now too.